### DISCUSSION TITLE
hex rgba?

### DISCUSSION THREAD

**Discussion by anuramat** [eylles/pywal16#73] [Q&A]
is there a way to get `#rrggbbaa`? trying to rice out my mako

**Comment by anuramat** [comment]
`{somecolor.alpha_hex}`/`{alpha.alpha_hex}`/`{somecolor.hex_argb}`. wiki looks dead, found this in changelogs/code. :wave:

**Reply by eylles** [reply]
Ah yes, i gotta do a lot of wiki updates just have forgotten to do so.
